server/src/routes/productsRoutes.js

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import { ProductsController } from '../controllers/productsController.js';
+
+const productsRoutes = Router();
+const prodCtrl = new ProductsController();
+
+productsRoutes.get('/products/', prodCtrl.getAllProducts);
```

Comment:
These `/` at the end of route are unnecessary.

Reply:
Done